main.go

```diff
@@ -134,9 +80,31 @@ func newFn() *cli.App {
 
 	prepareCmdArgsValidation(app.Commands)
 
+	cli.HelpPrinter = func(w io.Writer, templ string, data interface{}) {
+		doStuff(w, templ, data, colour.Colours)
+	}
+
 	return app
 }
 
+func doStuff(out io.Writer, templ string, data interface{}, customFunc map[string]interface{}) {
+	funcMap := colour.Colours
```

Comment:
do you need to be  passing the color map in as an arg and also using as the default values here?

Reply:
https://github.com/urfave/cli/blob/master/help.go#L236

I overwrite this function which is used to customise the AppTemplate.